receiver/bindplaneauditlogs/logs.go

```diff
@@ -0,0 +1,236 @@
+// Copyright observIQ, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bindplaneauditlogs
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"sort"
+	"sync"
+	"time"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/plog"
+	"go.uber.org/zap"
+)
+
+const (
+	bindplaneTimeFormat = time.RFC3339
+)
+
+// Add a wrapper struct for the API response
+type apiResponse struct {
+	AuditEvents []AuditLogEvent `json:"auditEvents"`
+}
+
+type bindplaneAuditLogsReceiver struct {
+	cfg           Config
+	client        *http.Client
+	consumer      consumer.Logs
+	logger        *zap.Logger
+	cancel        context.CancelFunc
+	wg            *sync.WaitGroup
+	lastTimestamp time.Time
+	settings      component.TelemetrySettings
+}
+
+// newBindplaneAuditLogsReceiver returns a newly configured bindplaneAuditLogsReceiver
+func newBindplaneAuditLogsReceiver(cfg *Config, logger *zap.Logger, consumer consumer.Logs) (*bindplaneAuditLogsReceiver, error) {
+	return &bindplaneAuditLogsReceiver{
+		cfg:           *cfg,
+		consumer:      consumer,
+		logger:        logger,
+		wg:            &sync.WaitGroup{},
+		lastTimestamp: time.Now().UTC().Add(-cfg.PollInterval),
+	}, nil
+}
+
+func (r *bindplaneAuditLogsReceiver) Start(ctx context.Context, host component.Host) error {
+	client, err := r.cfg.ToClient(ctx, host, r.settings)
+	if err != nil {
+		return fmt.Errorf("failed to create HTTP client: %w", err)
+	}
+	r.client = client
+
+	ctx, cancel := context.WithCancel(ctx)
```

Comment:
This should be made with `context.Background` since the passed in context will get canceled shortly after this func is called.
```suggestion
	ctx, cancel := context.WithCancel(context.Background())
```